Makefile

```diff
@@ -47,7 +47,7 @@ endif
 
 front_end:
 	@echo "Building Front End..."
-	cd $(makefile_dir)/metabase/; yarn build && yarn build-static-viz
+	cd $(makefile_dir)/metabase/; export WEBPACK_BUNDLE=production && yarn build && yarn build-static-viz
```

Comment:
Strange, I didn't see anything about this in the [changelog](https://www.metabase.com/docs/latest/developers-guide/driver-changelog#metabase-0460).